cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,332 @@
+  --header "Authorization: Bearer ${API_KEY}"
+~~~
+
+~~~txt
+{
+  "status": "PENDING",
+  "x509_pem_cert": ""
+}
+~~~
+
+~~~txt
+{
+  "status": "IS_SET",
+  "x509_pem_cert": "-----BEGIN CERTIFICATE-----\nMIIDfzCCAmagAwIBAgIBADANBgkqhkiG9w0BAQ0FADBZMQswCQYDVQQGEwJ1czEL\nMAkGA1UECAwCV0ExDTALBgNVBAoMBHRlc3QxDTALBgNVBAMMBHRlc3QxEDAOBgNV\nBAcMB1NlYXR0bGUxDTALBgNVBAsMBHRlc3QwHhcNMjMwMzE2MjMyNTMxWhcNMjQw\n
+...\n-----END CERTIFICATE-----",
+}
+~~~
+
+</section>
+<section class="filter-content" markdown="1" data-scope="tf">
+
+
+Add the following block to your terraform configuration and apply:
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+resource "cockroach_client_ca_cert" "yourclustername" {
+  id = cockroach_cluster.example.id
+  x509_pem_cert = file("cockroach_client_ca_cert.json")
+}
+~~~
+</section>
+
+## Update the certificate authority (CA) certificate for a dedicated cluster
+
+Replace the CA certificate used by your cluster for certificate-based client authentication.
+
+{{site.data.alerts.callout_success}}
+Managing the certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="api">Using the API</button>
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request PATCH \
+  --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+  --header "Authorization: Bearer ${API_KEY}" \
+  --header 'content-type: application/json' \
+  --data "@cockroach_client_ca_cert.json"
+~~~
+
+~~~txt
+200 OK
+~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="tf">
+Update the `cockroach_client_ca_cert` block in your terraform configuration, pointing to the updated certificate JSON file, and apply:
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+resource "cockroach_client_ca_cert" "yourclustername" {
+  id = cockroach_cluster.example.id
+  x509_pem_cert = file("cockroach_client_ca_cert.json")
+}
+~~~
+</section>
+
+## Delete the certificate authority (CA) certificate for a dedicated cluster
+
+Remove the configured CA certificate from the cluster. Clients will no longer be able to authenticate with certificates signed by this CA certificate.
+
+{{site.data.alerts.callout_success}}
+Managing the certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role.
```

Comment:
Ditto for keeping this in sync

Reply:
sorry Matt, I missed this comment and now am not sure how to find which previous , comment you were referring to, do you recall the suggestion?